image/image_linux.go

```diff
@@ -467,7 +468,21 @@ func (s *imageSeeder) deriveInfoForLocalSnaps(f seedwriter.SeedAssertionFetcher,
 	return snaps, s.w.InfoDerived()
 }
 
-func (s *imageSeeder) downloadSnaps(snapsToDownload []*seedwriter.SeedSnap, curSnaps []*tooling.CurrentSnap) (downloadedSnaps map[string]*tooling.DownloadedSnap, err error) {
+func (s *imageSeeder) validationSetKeys(db *asserts.Database) ([]snapasserts.ValidationSetKey, error) {
```

Comment:
maybe instead of passing it in we should store the db on imageSeeder in start

Reply:
I like this suggestion, it definitely makes sense now